src/fiatExchanges/utils.tsx

```diff
@@ -255,20 +255,19 @@ export const filterLegacyMobileMoneyProviders = (
   )
 
   return activeProviders.filter((provider) =>
-    provider[selectedCurrency === CiCoCurrency.CUSD ? 'cusd' : 'celo'].countries.includes(
+    provider[selectedCurrency === CiCoCurrency.cUSD ? 'cusd' : 'celo'].countries.includes(
```

Comment:
I realize you aren't introducing this in this PR, but it seems weird that the function assumes `selectedCurrency` is either cUSD or CELO, but the type for that parameter doesn't reflect that assumption. Seems like the type should be `CiCoCurrency.cUSD | CiCoCurrency.CELO`